c2corg_ui/static/js/documentdetailsview.js

```diff
@@ -68,6 +89,47 @@ app.ViewDetailsController.prototype.openModal = function(selector) {
   this.modal_.open({animation: true, size: 'lg', template: template});
 };
 
+
+/**
+ * @param {Date} dateStart
+ * @param {Date} dateEnd
+ * @suppress {checkTypes} it told (found scope | null, required -> scope) at compile()(this.scope)...
+ * @export
+ */
+app.ViewDetailsController.prototype.displayDate = function(dateStart, dateEnd) {
+
+  //angular executes this function so many times it tells 'Error: 10 $digest() iterations reached. Aborting' so
+  // with this quick n dirty method there is no error. Any idea how to do it in the pro way ?
```

Comment:
> We have to find why this function is called many times!

This is the normal behavior for bound values. In its digest cycle Angular checks if the value has changed. Because this function returns a new object on every call, Angular assumes that something has changed and runs the digest again. You could cache the object that you are returning and only create a new one when dateStart or dateEnd have changed. One-time binding could also work.

Maybe it would be even better to create a separate directive for this part. This would make it easier to reuse it elsewhere.